admin/templates/configuration/etc/xrdssi.cnf

```diff
@@ -29,8 +29,8 @@ location = {{QSERV_DATA_DIR}}/mysql
 [scheduler]
 
 # Thread pool size
-# thread_pool_size = 10
-thread_pool_size = 20
+# thread_pool_size = 20
+thread_pool_size = 12
```

Comment:
I've seen these number change a lot. They shouldn't be constants but come from a config file.

Reply:
This change is already in the configuration file :-) ..unless I misinterpreted your comment.

Reply:
Ah, I misinterpreted the file type. It is a config file, OK.